lib/node_modules/@stdlib/math/base/special/roundsd/README.md

```diff
@@ -85,36 +85,106 @@ logEachMap( 'x: %0.4f. y: %d. Rounded: %0.4f.', x, 5, roundsd );
 
 <!-- /.examples -->
 
-<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
-
-<section class="related">
+<!-- C interface documentation. -->
 
 * * *
 
-## See Also
+<section class="c">
+
+## C APIs
 
--   <span class="package-name">[`@stdlib/math/base/special/ceilsd`][@stdlib/math/base/special/ceilsd]</span><span class="delimiter">: </span><span class="description">round a numeric value to the nearest number toward positive infinity with N significant figures.</span>
```

Comment:
While this will get reinserted through automation, in general, for existing packages, we shouldn't be manually removing the list of related packages. There is not a reason for doing so.